### DISCUSSION TITLE
How did you create it? or rather the process and knowledge behind it?

### DISCUSSION THREAD

**Discussion by xkaku1** [TheGlockMisc/eivom-cli#2] [Q&A]
What is the scripting process or process of creating this? How can a beginner like me make one for myself? How do you scrape? Very exciting script you got :D

**Comment by Rendevior** [comment]
> What is the scripting process or process of creating this? How can a beginner like me make one for myself? How do you scrape? Very exciting script you got :D

TL;DR: Just Practice.

It is hard to explain in one comment but.
If you study on how it works by reading manuals and applying it, that will give you idea and it will help you to understand reading it.

First off i prefer starting on basics like Deleting/Replacing contents of file using **(AWK/SED)**.
You must practice **Regular Expression(REGEX)** for searching Matches and which Line/Columns you want to replace or get.

*sample.txt*
```
Hello Ipsum
```
Ex.. `sed 's/Ipsum/World/g' sample.txt` <- this will replace Ipsum to World

*Output*
```
Hello World
```

Conditionals/Loop Statements **(IF, WHILE, FOR, etc..)** are so important in Bash Scripting that could help you in handling errors and many more.

*Example Syntax:*
```
# if the value of Hello is not 1, the statement is False
Hello="1"

if [[ $Hello == "1" ]]; then
    echo "True, the Hello variable is equal to 1"
else
    echo "False, Hello variable is not equal to 1"
fi
```

In terms of scraping the sites, you should check meta tags and **HTTP RESPONSE INFO** also the Tokens to avoid rejecting the Request. (you may not understand this one at this point but i will mention it)

That's pretty much it, also not to mention that RegEx is so Helpful in other language too. 
[Also Bash Scripting introduces me in Programming (Python, Ruby, etc..), in my experience]

I'm not an intelectual person this is just based on my Experience/Learnings on the past few months.

(Sorry if my english is bad, I'm just using translator. I'm not using Japanese on this platform as much as possible)
